app/src/main/java/com/hedvig/app/ui/compose/composables/Transitions.kt

```diff
@@ -0,0 +1,22 @@
+package com.hedvig.app.ui.compose.composables
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.AnimatedVisibilityScope
+import androidx.compose.animation.ExperimentalAnimationApi
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.runtime.Composable
+
+@ExperimentalAnimationApi
```

Comment:
Do we *need* to propagate this annotation to the callers? It's just us consuming this composable anyway, so we are already fine with optin into it, maybe we should just optin here and not let the callers to have to add the extra annotation.